web/src/features/fireWeather/slices/parseModelValuesSlice.ts

```diff
@@ -0,0 +1,33 @@
+import { ModelRun, ModelValue } from 'api/modelAPI'
+import { isNoonInPST } from 'utils/date'
+
+export const parseModelValuesSlice = (
```

Comment:
Can you change this name as well as the filename? this is technically a helper function, not a slice